docs/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ add_subdirectory(HPCCSystemAdmin)
 add_subdirectory(IMDB)
 add_subdirectory(InstantCloud)
 add_subdirectory(Installing_and_RunningTheHPCCPlatform)
-add_subdirectory(RDDERef)
+add_subdirectory(RoxieReference)
 add_subdirectory(RunningHPCCinaVirtualMachine)
 add_subdirectory(ECLPlayground)
 add_subdirectory(ECLPluginForEclipse)
```

Comment:
Why is the subdirectory named RoxyRef_Mods? RoxieRefMods would be ok, but not Roxy